04-Docker.md

```diff
@@ -1,62 +1,76 @@
 # Part IV: Docker
 
-## Objectives: 
-By the end of this module, the SRE will: 
-- Describe the directives from a sample dockerfile 
-- Explain layers and the docker cache 
-- Start a docker container that runs ubuntu linux 
+## Objectives:
+By the end of this module, the SRE will:
+- Describe the directives from a sample dockerfile
+- Explain layers and the docker cache
+- Start a docker container that runs ubuntu linux
 - Describe the following container concepts:
   - base image
   - single pid
-  - command vs entrypoint
+  - `CMD` vs `ENTRYPOINT`
   - logs
   - volumes
   - layers and ordering
   - caching
   - build
   - push
 
-## Deliverables 
-1. Complete the project described below. Answer the questions, then schedule a time to meet with your mentor and discuss the what you've built.  
+## Deliverables
+Complete the project described below. Answer the questions, then schedule a time to meet with your mentor and discuss the what you've built.  
 
-### Docker 
+### Docker
+  1. Setup
+  - https://www.youtube.com/watch?v=JprTjTViaEA
```

Comment:
Please move this links so that are bullet points under *Install Docker* or have a heading that says *Resources*

Reply:
Done!